xc/device.ml

```diff
@@ -2983,6 +2986,8 @@ module Backend = struct
           |> List.concat |> List.concat
         in
 
+        let disks'' = Config.DISK.extra_args @ disks' in
```

Comment:
You can add this above line 2985, like this:
```
[ List.map ...
; Config.DISK.extra_args
; disks_other ... ]
```

It will then get concatenated before all the nvme disks, and we don't need a second variable.

Reply:
OK